addons/website/static/src/scss/website.scss

```diff
@@ -1410,3 +1410,11 @@ $ribbon-padding: 100px;
     // avoid having to create a new scss file in a stable version.
     align-self: flex-start;
 }
+
+.modal-open {
+    .o_bottom_fixed_element {
+        // prevent bottom fixed element (e.g. the livechat button)
+        // from hidding the buttons or any important part of the modal.
+        transform: translateY(100%);
```

Comment:
Same comment: should use the same class, not hiding with translate. What if a theme moves the livechat button 3px higher? Once hidden with this, they would still be 3px visible.